### PR TITLE
My profile ks

### PR DESCRIPTION
**Pull request by kovacssandorUI**
Ready my profile update, with change password button. If it is fine, can you merge to master.

### CODE REVIEW

File: app/script/controllers/my.profile.controller.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                $scope.myProfileUpdate=function () {
+
+                    apiService.myProfileUpdate($scope.myProfile)
+                        .then(function (response) {
+                            $scope.myProfile=response.data;
+                        })
+                };
+
+            $scope.checkPhone = function(data) {
+                var re = new RegExp(/^\(?(\d{2})\)?[- ]?(\d{3})[- ]?(\d{4})$/);
```
Comment (szabo-tamas-intland) [inline app/script/controllers/my.profile.controller.js:21]: I don't think people put parentheses around '06', we don't have to check it, and also, you don't use the groups you matched that's unnecessary too.

File: app/index.html [hunk 2]
```diff
@@ -19,7 +20,7 @@
                 <input id="searchfield" type="text" class="form-control" placeholder="Search" ng-model="search"/>
             </div>
             <div class="col-md-5 col-sm-5" id="left-text">
-                <a class=" glyphicon glyphicon-user" href="#/user/" ng-if="token">Profile</a>
+                <a class=" glyphicon glyphicon-user" href="#/my_profile" ng-if="token">Profile</a>
```
Comment (szabo-tamas-intland) [inline app/index.html]: Please use kebab-case for links (my-profile)

File: app/script/controllers/my.profile.controller.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                if (!re.test(data)) {
+                    return "Your phone number should look like: 'xx-xxx-xxxx'";
+                }
+            };
+            $scope.rights=[ {id:1,name:"ADMIN"},
+                            {id:2, name:"CUSTOMER"},
+                            {id:3, name: "WAREHOUSE_OWNER"}];
+
+            $scope.removeMyProfile=function () {
+                    console.log("remove");
```
Comment (szabo-tamas-intland) [inline app/script/controllers/my.profile.controller.js]: Please remove

File: app/script/controllers/my.profile.controller.js (hunk 1 above)
Comment (kovacssandorUI) [reply app/script/controllers/my.profile.controller.js:21]: So, here right the(70)-111-1111, but also right this:70-111-111, and right this 70111111 too. Every format is ok, just have nine decimal number. 
What will be the expected format?

File: app/index.html (hunk 2 above)
Comment (kovacssandorUI) [reply app/index.html]: fixed

File: app/script/controllers/my.profile.controller.js (hunk 3 above)
Comment (kovacssandorUI) [reply app/script/controllers/my.profile.controller.js]: I removed.

File: app/script/controllers/my.profile.controller.js (hunk 1 above)
Comment (szabo-tamas-intland) [reply app/script/controllers/my.profile.controller.js:21]: It's ok, leave it as is :)

**Pull request merged**